vitest.base.unit.config.ts

```diff
@@ -1,6 +1,6 @@
 import path from "node:path";
 import {defineConfig} from "vitest/config";
-const __dirname = new URL(".", import.meta.url).pathname;
+const currentDir = new URL(".", import.meta.url).pathname;
```

Comment:
I’m partial to leaving this as `__direnv`.  I dont feel strongly about it but it’s sort of the “node way” to name that

Reply:
Yes I had that initially as `__dirname` here and other places as well, but feel like a old pattern now.

Reply:
For consistency with the rest of the codebase `__dirname` is better, but no strong opinion here